ietf/api/views.py

```diff
@@ -192,7 +192,7 @@ def err(code, text):
             except ValueError as e:
                 return err(400, "Unexpected POST data: %s" % e)
             response = "Accepted, New registration" if created else "Accepted, Updated registration"
-            if User.objects.filter(username=email).exists() or Email.objects.filter(address=email).exists():
+            if User.objects.filter(username__iexact=email).exists() or Email.objects.filter(address=email).exists():
```

Comment:
With the existing implementation, the email address being submitted actually originates from Datatracker so the clause is not really needed. Unfortunately the the send_account_creation_email is generic, so there is no way to confirm this based on messages sent.

Reply:
Better to leave it this way in case other things get out of sync.